ci/scripts/java_full_build.sh

```diff
@@ -60,10 +57,6 @@ mvn clean \
     -DdescriptorId=source-release
 
 # copy all jar, zip and pom files to the distribution folder
-find . \
-     "(" -name "*-javadoc.jar" -o -name "*-sources.jar" ")" \
-     -exec echo {} ";" \
-     -exec cp {} $dist_dir ";"
```

Comment:
This is covered by the line below?

Reply:
This is covered by the `install` phase which also includes the previous phases like compile or package. The same steps are already declared in the apache parent pom when using the `arrow-release` profile.
- `assembly:single` was used to generate a single artifact with the whole tree and is handled by the parent plugin with the [`source-release-assembly`](https://github.com/apache/maven-apache-parent/blob/36c52ca00b390762f2571fadf4802497b7b80cab/pom.xml#L441) execution in the parent pom (I should also remove the `-DdescriptorId=` property as well I guess)
- `source:jar` is handled by [`attach-sources`](https://github.com/apache/maven-apache-parent/blob/36c52ca00b390762f2571fadf4802497b7b80cab/pom.xml#L462) execution bound by default to `package` phase 
- `javadoc:jar` is handled by the [`attach-javadocs`](https://github.com/apache/maven-apache-parent/blob/36c52ca00b390762f2571fadf4802497b7b80cab/pom.xml#L474) execution bound by default to the `package` phase

Reply:
Ah sorry, might I misread your question. If you're asking about removing the `find` command, yes, it's because it's handled by the second `find` command which will get the single assembly, the sources and javadoc jars from the local repository and copy them to the `$dist_dir` directory